landing_page.php

```diff
@@ -0,0 +1,12 @@
+<html> 
+    <head>
+    <script defer>
+        function login() {
+            window.location.href = 'login.php';
+        }
+    </script>
+    </head>
+    <body>
+        <button onclick="login()">login with spotify</button>
```

Comment:
You can leave this a button if you want, but if it's just a redirect, I'd probably just make it an a tag styled however you want it to look